### ISSUE TITLE
Spam on Mapknitter

### ISSUE THREAD

**Issue by steviepubliclab** [publiclab/mapknitter#246]
### What happened just before the problem occurred
There's a lot of spam posted on Mapknitter lately. Some are posted with images in the maps, while others just have text in their "about" box posted next to the maps. On Publiclab.org we have a group that helps to deal with spam on the site through a moderation system that allows for people to review post from first time posters before they go live. We don't have a system like this on Mapknitter, but I think spam is getting increasingly worse on that site. It would be great to brainstorm some ideas for how this spam could be handled!

**Comment by jywarren** [comment]
Yeah, that is a problem. Were most anonymous, or actually registered on PublicLab.org? 

different ideas:

1. have an email sent when new maps appear, so we can keep an eye out better
2. recruit a second moderators group? 
3. holding maps for moderation, though you'd be able to edit/view them yourself immediately

Are the images people upload actually placed somewhere? Could we not show images if they haven't been moved around on the map at least a bit? Although that might not stop people for long.

**Comment by steviepubliclab** [comment]
Copying and replying back in italics:: 

Yeah, that is a problem. Were most anonymous, or actually registered on PublicLab.org? _It varied, a number of them were associated with profiles_

different ideas:

have an email sent when new maps appear, so we can keep an eye out better _this sounds good to me_
recruit a second moderators group? _might be hard to manage two groups, get's a bit confusing, but I see the value in recruiting specifically from the grassroots mapping group for this one, and that group might not be as interested in the PL site moderation_ 
holding maps for moderation, though you'd be able to edit/view them yourself immediately _I like this_
Are the images people upload actually placed somewhere? _a number of them looked like they were placed_ Could we not show images if they haven't been moved around on the map at least a bit? Although that might not stop people for long.

**Comment by SidharthBansal** [comment]
Part of #300

**Comment by SidharthBansal** [comment]
Hi, I am little new to spam moderation system at PL. I found some irrelevant posts
https://mapknitter.org/maps/testmap28
https://mapknitter.org/maps/test--175
https://mapknitter.org/maps/test-map--20
Can we delete them from the website?

**Comment by jywarren** [comment]
Hi Sidharth, there is a Archive/Spam function shown on the dashboard to
admin users. We need to make it easier to make admin/mod users, which could
be part of the "sharing state from PL.org" project (i.e. just look to role
in PL.org). We could also make an "archive/spam" button on individual map
pages, too. Make sense?

**Comment by SidharthBansal** [comment]
Cool thanks
